README.md

```diff
@@ -90,11 +76,13 @@ Parameters:
 
 Returns a `FirebaseDeviceStore`.
 
-#### `FirebaseDeviceStore.signOut(): void`
+#### `FirebaseDeviceStore.signOut(): Task<Void>`
 
 Indicate to the DeviceStore that the user is about to sign out, and the current device token should be removed.
```

Comment:
Nit: s/to sign out, and the/to sign out and that the/

Reply:
Changed